src/partitioner/local/MetisPartitioner.cpp

```diff
@@ -385,6 +391,139 @@ void MetisPartitioner::createPartitionFiles(std::map<int, int> partMap) {
         this->sqlite.runUpdate(sqlStatement);
         dbLock.unlock();
     }
+
+    //Below code segment will create the composite central stores which uses for aggregation
+
+    if (nParts > 4) {
```

Comment:
Here also we should use a constant instead of 4.

Reply:
Used a constant

Reply:
Done